tests/test_models/test_tbats.py

```diff
@@ -4,8 +4,10 @@
 
```

Comment:
May you also add test that we get the correct set of components depending on the parameters

Reply:
Added test